src/components/carousel/__tests__/index.spec.tsx

```diff
@@ -54,12 +51,13 @@ describe('Carousel render tests', () => {
 
   describe('onScroll', () => {
     it('should trigger onScroll from the second scroll', async () => {
-      const driver = new CarouselDriver({component: <TestCase/>, testID});
+      const renderTree = render(<TestCase/>);
+      const driver = NewCarouselDriver({renderTree, testID});
 
-      await driver.scroll(Constants.screenWidth); //NOTE: first scroll doesn't fire onScroll
+      await driver.scroll({contentOffset: {x: Constants.screenWidth, y: 0}}); //NOTE: first scroll doesn't fire onScroll
```

Comment:
Please verify the "first call" is still needed

Reply:
Yes, it's still needed.